azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -173,7 +173,7 @@ private void setupMultiExecutorMode() {
       new QueueProcessorThread(azkProps.getBoolean(
         AZKABAN_QUEUEPROCESSING_ENABLED, true), azkProps.getLong(
         AZKABAN_ACTIVE_EXECUTOR_REFRESHINTERVAL_IN_MS, 1000), azkProps.getInt(
-        AZKABAN_ACTIVE_EXECUTOR_REFRESHINTERVAL_IN_NUM_FLOW, 1000));
+        AZKABAN_ACTIVE_EXECUTOR_REFRESHINTERVAL_IN_NUM_FLOW, 10));
```

Comment:
I think the INTERVAL part is causing the confusion.

How about AZKABAN_ACTIVE_EXECUTOR_REFRESH_BY_NUM_FLOW?